presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java

```diff
@@ -31,4 +36,31 @@ public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
     {
         super(connectorId, config, "\"", new SQLServerDriver());
     }
+
+    @Override
+    public void commitCreateTable(JdbcOutputTableHandle handle)
+    {
+        StringBuilder sql = new StringBuilder()
+                .append("sp_rename ")
```

Comment:
Why?
Why not overriding `beginCreateTable`, then having `commitCreateTable` straightforward - simply calling `commit`?